parsec/core/types/manifest.py

```diff
@@ -217,6 +249,27 @@ def from_remote(cls, remote: RemoteManifest) -> "LocalManifest":
         elif isinstance(remote, RemoteUserManifest):
             return LocalUserManifest.from_remote(remote)
 
+    def to_remote(self) -> RemoteManifest:
+        raise NotImplementedError
+
+    def match_remote(self, remote_manifest: RemoteFileManifest) -> bool:
+        reference = self.to_remote(
+            author=remote_manifest.author, timestamp=remote_manifest.timestamp
+        )
+        return reference.evolve(version=remote_manifest.version) == remote_manifest
+
+    # Debugging
+
+    def asdict(self):
+        dct = attr.asdict(self)
+        dct.pop("base")
+        props = "base_version", "is_placeholder", "created"
```

Comment:
pas mieux de faire directement `for name in ("base_version", "is_placeholder", "created"):` ?

Reply:
90412fea8e31e328c8fcd54fc5d66d230e3a60e3